sdks/builds/runtime.mk

```diff
@@ -142,7 +142,7 @@ _cross-runtime_$(1)_CONFIGURE_FLAGS= \
 
 .stamp-$(1)-$$(CONFIGURATION)-configure: | $$(if $$(IGNORE_PROVISION_LLVM),,provision-$(5))
 
-$$(TOP)/sdks/builds/$(1)-$$(CONFIGURATION)/$(3).h: .stamp-$(1)-$$(CONFIGURATION)-configure $$(TOP)/tools/offsets-tool/MonoAotOffsetsDumper.exe | configure-$(4)
+$$(TOP)/sdks/builds/$(1)-$$(CONFIGURATION)/$(3).h: .stamp-$(4)-$$(CONFIGURATION)-configure .stamp-$(1)-$$(CONFIGURATION)-configure $$(TOP)/tools/offsets-tool/MonoAotOffsetsDumper.exe | configure-$(4)
```

Comment:
This is unnecessary: the chain on dependency should be `configure-$(4)` -> `.stamp-$(4)-configure` -> `.stamp-$(4)-$$(CONFIGURATION)-configure`. You can see that at https://github.com/mono/mono/blob/a4d0e665e190e551b043558d7c1d26bf5bcf705a/sdks/builds/Makefile#L88 and https://github.com/mono/mono/blob/a4d0e665e190e551b043558d7c1d26bf5bcf705a/sdks/builds/runtime.mk#L70 respectively. If that chain of dependency is broken for the `wasm` targets, the `wasm` targets should be themselves be fixed.

Reply:
Ok, I'll change to be `.stamp-$(4)-configure`.
This dep is needed otherwise building the win cross compiler by itself doesn't work.

Reply:
This dep is requires because the offsets file is generated by the configure of the runtime configuration.

IE, `configure-wasm-runtime-release` produces the offset needed by `build-wasm-crosswin-release`